docs/outlook/privacy-and-security.md

```diff
@@ -42,19 +42,27 @@ The following figure shows the four levels of permissions and describes the capa
 
 ![4-tier permissions model for mail apps schema v1.1](../images/add-in-permission-tiers.png)
 
-## AppSource: add-in integrity
+## AppSource: Add-in integrity
 
 [AppSource](https://appsource.microsoft.com) hosts add-ins that can be installed by end users and administrators. AppSource enforces the following measures to maintain the integrity of these Outlook add-ins:
 
 - Requires the host server of an add-in to always use Secure Socket Layer (SSL) to communicate.
 
-- Requires a developer to provide proof of identity, a contractual agreement, and a compliant privacy policy to submit add-ins. 
+- Requires a developer to provide proof of identity, a contractual agreement, and a compliant privacy policy to submit add-ins.
 
 - Archives add-ins in read-only mode.
 
 - Supports a user-review system for available add-ins to promote a self-policing community.
 
-## End users: privacy and performance concerns
+## Optional connected experiences
+
+Users can turn off [optional connected experiences in Office](/deployoffice/privacy/optional-connected-experiences) desktop and mobile clients. For Outlook add-ins, generally this means that users who turn off the **Optional connected experiences** setting can no longer get add-ins through these clients. However, certain Microsoft add-ins that are considered essential or business-critical, and add-ins deployed by an organization's IT admin are still available. Since the setting doesn't affect the availability of add-ins and the Office Store in Outlook on the web, the user can use the web client to [manage their add-ins](https://support.microsoft.com/office/8f2ce816-5df4-44a5-958c-f7f9d6dabdce), including admin-deployed ones.
```

Comment:
should we mention that the get Add-ins button will NOT be available in Windows and MAc? but they can be managed in OWA.  In mobile they will have the get add-ins experience but just with add-ins deployed via centralized deployemnt (admin-managed)